client/src/components/VideoForm.js

```diff
@@ -0,0 +1,108 @@
+import { useState } from "react";
+const VideoForm = ({ getAllVideos }) => {
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const [backendMessage, setBackendMessage] = useState("");
```

Comment:
What's the backend message? Can you think of a more clear name for this, or add a comment explaining what it's for?